Code/Mantid/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py

```diff
@@ -157,8 +157,9 @@ def category(self):
         return 'Diffraction;PythonAlgorithms'
 
     def summary(self):
-        return "This Python algorithm performs the operations necessary for the reduction of diffraction data from the Osiris instrument at ISIS \
-              into dSpacing, by correcting for the monitor and linking the various d-ranges together."
+        return "This Python algorithm performs the operations necessary for the reduction of diffraction data \
+                from the Osiris instrument at ISIS \
+                into dSpacing, by correcting for the monitor and linking the various d-ranges together."
 
```

Comment:
Won't the indentation here end up being included in the error string? Maybe having a separate string on each line, and joining them with a `+` would be better.